src/test/scala/com/microsoft/hyperspace/TestUtils.scala

```diff
@@ -24,4 +26,24 @@ object TestUtils {
     result.state = state
     result
   }
+
+  /**
+   * Split path into its segments and return segment names as a sequence.
+   * For e.g. a path "file:/C:/d1/d2/d3/f1.parquet" will return
+   * Seq("file:/C:/", "d1", "d2", "d3", "f1.parquet")
```

Comment:
Looks like this function will return the reverse of this?

Reply:
YES! thanks... I will fix it.

Reply:
thanks @imback82 , done